e2e_tests/tests/deploy/test_local.py

```diff
@@ -103,7 +103,29 @@ def resource_manager(
 def test_cluster_down() -> None:
     name = "test_cluster_down"
 
-    with resource_manager(Resource.CLUSTER, name, {}, ["no-gpu"]):
+    with resource_manager(Resource.CLUSTER, name, {}, ["no-gpu", "enterprise-edition"]):
```

Comment:
Depending on what the eventual tests will look like, maybe best is to prefer [pytest.mark.parametrize](https://docs.pytest.org/en/latest/how-to/parametrize.html#parametrize-basics) to copy/pasting the test body.